lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -473,15 +473,13 @@ def cmd_search(*args)
               }
             else
               print_line(tbl.to_s)
-              print_status("Using #{used_module}") if used_module
+              index_usage = "use #{@module_search_results.length - 1}"
+              index_info = "info #{@module_search_results.length - 1}"
+              name_usage = "use #{@module_search_results.last.fullname}"
 
-              if @module_search_results.length > 1
-                index_usage = "use #{@module_search_results.length - 1}"
-                index_info = "info #{@module_search_results.length - 1}"
-                name_usage = "use #{@module_search_results.last.fullname}"
+              print("Interact with a module by name or index. For example %grn#{index_info}%clr, %grn#{index_usage}%clr or %grn#{name_usage}%clr\n\n")
 
-                print("Interact with a module by name or index, for example %grn#{index_info}%clr, %grn#{index_usage}%clr or %grn#{name_usage}%clr\n\n")
```

Comment:
I realise you're just moving this print statement but should it be `print_status` rather than `print` I'm not actually sure on the semantics here, so I could well be wrong but some clarification would be nice

Reply:
@dwelch-r7 The screenshot in the PR description shows the differences between `print` and `print_status`
![image](https://user-images.githubusercontent.com/60357436/92478277-d6abc500-f1d9-11ea-96aa-23952e1ae3b9.png)

In this scenario, `print_status` comes with an extra `[*]` at the start that isn't required for this message to the user.

Reply:
gotcha, cool cheers